core/behavior/__init__.py

```
@@ -0,0 +1,13 @@
+"""
+行为画像模块 — 纯统计，零 LLM 依赖
+
+提供:
+- MessageCleaner: 过滤代码块、转发、引用
+- LiwcAnalyzer: LIWC 词频统计（情绪/代词/认知）
+- BehaviorBaseline: 长期行为基线计算
+- BehaviorProfileManager: 全局管理器，接入消息流
+- get_soul_offset(): soul_state 相对基线偏移（等 soul_state 重构时接入）
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)